### PR TITLE
[PAE 110] Fix pylint errors

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description
Activate pylint and fix errors.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -46,8 +46,8 @@ jobs:
             . venv/bin/activate
             coverage run --source="." manage.py test
             coverage report --fail-under=83 -m
-            # pycodestyle ./openedx_external_enrollments
-            # pylint ./openedx_external_enrollments --rcfile=./setup.cfg
+            pycodestyle ./openedx_external_enrollments
+            pylint ./openedx_external_enrollments --rcfile=./setup.cfg
             # isort --check-only --recursive --diff ./openedx_external_enrollments
```
Comment (Squirrel18) [inline .circleci/config.yml:51]: What's wrong with isort? Why didn't you enable it? @andrey-canon

File: .circleci/config.yml [hunk 2]
```diff
@@ -46,8 +46,8 @@ jobs:
             . venv/bin/activate
             coverage run --source="." manage.py test
             coverage report --fail-under=83 -m
-            # pycodestyle ./openedx_external_enrollments
-            # pylint ./openedx_external_enrollments --rcfile=./setup.cfg
+            pycodestyle ./openedx_external_enrollments
```
Comment (Squirrel18) [inline .circleci/config.yml:49]: Can we include these commands in the Makefile? @andrey-canon

File: openedx_external_enrollments/edxapp_wrapper/backends/openedx_permissions_i_v1.py [hunk 3]
```diff
@@ -1,6 +1,5 @@
 """ Backend abstraction """
-from openedx.core.lib.api.permissions import (
-    # pylint: disable=unused-import
+from openedx.core.lib.api.permissions import (  # pylint: disable=unused-import, import-error
```
Comment (Squirrel18) [inline openedx_external_enrollments/edxapp_wrapper/backends/openedx_permissions_i_v1.py:2]: Is the unused-import necessary? @andrey-canon

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (andrey-canon) [reply .circleci/config.yml:51]: isort is a different pr

File: openedx_external_enrollments/edxapp_wrapper/backends/openedx_permissions_i_v1.py (hunk 3 above)
Comment (andrey-canon) [reply openedx_external_enrollments/edxapp_wrapper/backends/openedx_permissions_i_v1.py:2]: pylint says yes

File: .circleci/config.yml (hunk 2 above)
Comment (andrey-canon) [reply .circleci/config.yml:49]: not, why did you suggest that in the pr where the tests are included?

Comment (Squirrel18) [reply .circleci/config.yml:49]: Good question, I didn't notice. @andrey-canon

File: openedx_external_enrollments/edxapp_wrapper/backends/openedx_permissions_i_v1.py (hunk 3 above)
Comment (Squirrel18) [reply openedx_external_enrollments/edxapp_wrapper/backends/openedx_permissions_i_v1.py:2]: Ummm, ok.

File: .circleci/config.yml (hunk 1 above)
Comment (Squirrel18) [reply .circleci/config.yml:51]: Any particular reason to include it in other PR? @andrey-canon

Comment (andrey-canon) [reply .circleci/config.yml:51]: too many changes

Comment (Squirrel18) [reply .circleci/config.yml:51]: Ok.

File: .circleci/config.yml (hunk 2 above)
Comment (Squirrel18) [reply .circleci/config.yml:49]: So let's change it in a subsequent PR, do you agree? @andrey-canon

**Review by Squirrel18 — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (andrey-canon) [reply .circleci/config.yml:49]: ok

**Pull request merged**